image-loader/conf/routes

```diff
@@ -7,8 +7,6 @@ GET     /images/project/:imageId    controllers.ImageLoaderController.projectIma
 GET     /uploadStatus/:imageId      controllers.UploadStatusController.getUploadStatus(imageId: String)
 POST    /uploadStatus/:imageId      controllers.UploadStatusController.updateUploadStatus(imageId: String)
 
-
-
```

Comment:
Noise, please remove.

Reply:
those are extra lines

Reply:
My point here is that this file isn't actually modified (semantically) but it has been modified making the PR look untidy (as it is making unnecessary changes).

Reply:
yup makes sense 👌🏾